pkg/plugin/conftest/plugin.go

```diff
@@ -166,13 +166,7 @@ func (p *plugin) GetScanJobSpec(ctx starboard.PluginContext, obj client.Object)
 					},
 				},
 			},
-			SecurityContext: &corev1.PodSecurityContext{
-				RunAsUser:  pointer.Int64Ptr(1000),
```

Comment:
- This user is not defined in Conftest image
- On OpenShift Container Platform it blocks scan Job from running